pkg/cli/restic.go

```diff
@@ -47,15 +47,32 @@ type Snapshot struct {
 	Gid      int       `json:"gid"`
 }
 
-func (w *ResticWrapper) ListSnapshots() ([]Snapshot, error) {
+func (w *ResticWrapper) ListSnapshots(snapshotIDs []string) ([]Snapshot, error) {
 	result := make([]Snapshot, 0)
-	args := w.appendCacheDirFlag([]interface{}{"snapshots", "--json"})
+	args := w.appendCacheDirFlag([]interface{}{"snapshots", "--json", "--quiet"})
 	args = w.appendCaCertFlag(args)
 
+	if snapshotIDs != nil {
```

Comment:
don't need to perform a nil check here
https://caretdashcaret.com/2017/04/16/iterating-over-nil-slices-in-go/

Reply:
Ok. refactored.